### DISCUSSION TITLE
Issue installing libmagic

### DISCUSSION THREAD

**Discussion by Ragav86** [Unstructured-IO/unstructured#615] [Q&A]
When I run a command to install libmagic in windows, I get PackagesNotFoundError.

How to resolve this issue for Windows 10 OS?

 conda install -c conda-forge libmagic
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are not available from current channels:

  - libmagic

Current channels:

  - https://conda.anaconda.org/conda-forge/win-64
  - https://conda.anaconda.org/conda-forge/noarch
  - https://repo.anaconda.com/pkgs/main/win-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/win-64
  - https://repo.anaconda.com/pkgs/r/noarch
  - https://repo.anaconda.com/pkgs/msys2/win-64
  - https://repo.anaconda.com/pkgs/msys2/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.

**Comment by ccalvocm** [comment]
Is your issue solved with
 `pip install python-libmagic`
`pip install python-magic-bin`
 ?

**Comment by Ragav86** [comment]
Yes, I was able to resolve the issue.

Thanks.
________________________________

**Comment by ccalvocm** [comment]
@Ragav86 Marvelous!
